ruddock/modules/users/routes.py

```diff
@@ -26,17 +27,64 @@ def show_memberlist(search_type):
       memberlist=memberlist,
       search_type=search_type)
 
-@blueprint.route('/view/<username>')
+@blueprint.route('/view/user_id=<user_id>')
 @login_required()
-def view_profile(username):
-  """Procedure to show a user's profile and membership details."""
-  user_info = helpers.get_user_info(username)
-  offices = helpers.get_office_info(username)
+def view_member(user_id):
+  """Procedure to show a member's profile and membership details if they have
```

Comment:
i might be reading this wrong, but isn't this also if they _do_ have a user account on the site?

Reply:
Oh right, I forgot to change this back from when I still had view_profile around for viewing members with accounts.